QUEST_LV_0100.tsv

```diff
@@ -812,18 +812,18 @@ QUEST_LV_0100_20150317_000811	Who cherished such traitor? Don't you think there
 QUEST_LV_0100_20150317_000812	Unexpectedly, the status is fine.{nl}We should get ready to restore it.
 QUEST_LV_0100_20150317_000813	His monument is cased with a terrible magic because he was a traitor.{nl}We should use that magic instead.
 QUEST_LV_0100_20150317_000814	If we received support than we would have hired mecenaries.{nl}Studying about the traitor doesn't mean that we praise the traitor so i don't understand why the support has stopped.
-QUEST_LV_0100_20150317_000815	Thanks.{nl}Achat must have drawn to the power of the magic that is casted upon the monument.
+QUEST_LV_0100_20150317_000815	$Thanks.{nl}Achat must have been drawn to the magic power cast upon the monument.
 QUEST_LV_0100_20150317_000816	You look live a wood so it's curious to find out that you are interested in stone plates.
 QUEST_LV_0100_20150317_000817	I am sorry, but we can't find the one that we needed among the fragments you brought.{nl}I hope it's not already eaten.
-QUEST_LV_0100_20150317_000818	Don't you think we would be able to find the fragments that Treeambulo swallowed if we burn it?{nl}I have the woods that I stacked before so we push it into it while the woods are burning.
-QUEST_LV_0100_20150317_000819	Be careful.{nl}It will be dangerous if Treeambulo in flame runs into the camp.
+QUEST_LV_0100_20150317_000818	$Don't you think we would be able to find the stone pieces that a Tree Ambulo swallowed if we burn it?{nl}I have stacked woods there, so we can push it in when the wood is burning.
+QUEST_LV_0100_20150317_000819	$Be careful.{nl}It will be dangerous if a flaming Tree Ambulo runs into the camp.
 QUEST_LV_0100_20150317_000820	Treasure Hunter Ethan
 QUEST_LV_0100_20150317_000821	I am a treasure hunter and not a monster hunter.{nl}I don't want to add my gravestone here by touching it.
-QUEST_LV_0100_20150317_000822	ve Collected all the debris out of the deal is over Raymond. {nl} brought back the parcel to Raymond.
-QUEST_LV_0100_20150317_000823	Ethan told you to give it to me?{nl}Hmm. It will take sometime to restore it.{nl}
-QUEST_LV_0100_20150317_000824	Can you take the restored gravestone fragments to my assistant at Irody Square?{nl}I think something good will happen when we finish restoring it.
+QUEST_LV_0100_20150317_000822	$I've collected all the debris Raymond requested. {nl} Bring them back to Raymond.
+QUEST_LV_0100_20150317_000823	$Ethan told you to give this to me?{nl}Hmm. It will take sometime to restore it.{nl}
+QUEST_LV_0100_20150317_000824	$Can you take these restored gravestone fragments to my assistant at Irody Square?{nl}I think something good will happen when we finish restoring it.
 QUEST_LV_0100_20150317_000825	Epigraphist Smid
-QUEST_LV_0100_20150317_000826	This.. Our master did it!{nl}Give it to me.
+QUEST_LV_0100_20150317_000826	$This.. Our master did it!{nl}Can I have it?
```

Comment:
This.. -> This...
"Can I have it?" has a very different tone compared to "Give it to me", which is what the original says. The original says: "Give it to me, please."